src/plugins/codeStyle/styleFixes.ts

```diff
@@ -142,3 +147,21 @@ function replaceFunctionTokens(diagnostic: BsDiagnostic, token: string) {
         ]
     };
 }
+
+function addEolLast(diagnostic: BsDiagnostic): ChangeEntry {
+    return {
+        diagnostic,
+        changes: [
+            insertText(diagnostic.range.end, platform.toString() === 'win32' ? '\r\n' : '\n')
```

Comment:
Rather than assuming the line ending, you should just look at the line ending from the previous line. Sometimes projects will use \n on windows too.